[2023-12-13 17:13:33] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn inbound conn 99d430df88c82261 127.0.0.1:61227 failed checkpoint posthandshake: Connected to self
[2023-12-13 17:13:33] [warning] server.go:646 dpchain/p2p/server.(*Server).setupConn dyn dial conn 99d430df88c82261 127.0.0.1:20134 failed checkpoint posthandshake: Connected to self
[2023-12-13 17:13:33] [warning] peer.go:166 dpchain/p2p/server.(*Peer).run Peer cce471fc46e94029 127.0.0.1:20131: remote requested disconnect: Already connected

[2023-12-13 17:13:33] [warning] synchronization.go:171 dpchain/blockSync.(*SyncProtocal).handle Block synchronization failed: Protocal handshake with node (cce471fc46e9402940bb2bcc4fbcee724dfcc14e020f6362ec2ad827bf6c4ef7354165b9de64807d37caae4888c31a86c34b77594ee64be150612991f6e95b61) is failed, err:EOF

[2023-12-13 17:13:33] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-13 17:13:33] [warning] separator.go:127 dpchain/core/separator.(*Separator).handlePeer separator: 协议handshake失败,err:EOF

[2023-12-13 17:13:33] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-13 17:13:33] [warning] whisper.go:226 dpchain/p2p/whisper.(*Whisper).handlePeer 无法发送whisper初始化消息,err:EOF

[2023-12-13 17:13:33] [warning] peer.go:319 dpchain/p2p/server.(*Peer).startProtocols.func1 协议handshake, 协议Run函数错误,err:EOF

[2023-12-13 17:13:33] [warning] peer.go:208 dpchain/p2p/server.(*Peer).pingLoop heart beate, ping协程退出

